robocup_msgs/InterestPoint itP
---
bool success